test/conformance/walk.go

```diff
@@ -291,10 +259,6 @@ func (v *visitor) Visit(node ast.Node) (w ast.Visitor) {
 			totalConfTests++
 			v.emit(t.Args[0])
 			return nil
-		} else if v.isLegacyItCall(t) {
-			totalLegacyTests++
-			v.failf(t, "Using It() with manual [Conformance] tag is no longer allowed.  Use framework.ConformanceIt() instead.")
-			return nil
```

Comment:
The output still uses `totalLegacyTests`:
https://github.com/kubernetes/kubernetes/blob/6e3d21b7b8c6d1f6d6a615cc1e2c6a87f47e8391/test/conformance/walk.go#L329-L332

Should the variable and that output perhaps be removed completely?

Reply:
Oh, I missed that.
Yeah, that is a nice point.
I will do that.